package.json

```diff
@@ -28,7 +28,7 @@
     "lint:scss": "stylelint '**/*.{css,scss}'",
     "lint:twig": "composer twig-lint lint ./apps && composer twig-lint lint ./source",
     "lint": "npm run lint:js; npm run lint:scss; npm run lint:twig",
-    "fmt:js": "npm run lint:js -- --fix && prettier --write '**/*.{js,json,vue}'",
+    "fmt:js": "npm run lint:js -- --fix",
```

Comment:
Did the use here change with a new version? Or is this a fix? It's broken currently on AWN and I waaant it.

Reply:
`eslint-plugin-prettier` runs prettier on `eslint --fix`. We've technically been running prettier TWICE against all our js/scss/css/vue this whole time!